src/iso19111/operation/parammappings.cpp

```diff
@@ -829,6 +829,10 @@ static const MethodMapping projectionMethodMappings[] = {
 
     {PROJ_WKT2_NAME_METHOD_QUADRILATERALIZED_SPHERICAL_CUBE, 0,
      "Quadrilateralized_Spherical_Cube", "qsc", nullptr, paramsNatOrigin},
+    
+    {PROJ_WKT2_NAME_METHOD_S2, 0,
+     "S2", "s2", nullptr, paramsNatOrigin},
```

Comment:
paramsNatOrigin doesn't include sUVtoST. I would just remove that definition for now

Reply:
Okay, should I remove those two lines from `parammappings.cpp` ?

Reply:
> Can you elaborate slightly on how these definitions are used in parameter parsing,

They are only needed when converting between PROJ.4 string style and WKT.  But WKT has not support for string values for a projection parameter, so having a proper WKT definition for S2 isn't trivial. When there's no projection method mappings, PROJ will generate something that is conformant to WKT formalism, but only interoperable in the PROJ universe, which is probably good enough for S2.

Reply:
So, if, for example, someone wanted to generate GeoTIFFs in S2 coordinates via GDAL, it would be beneficial to have the correct parameter mappings available to stuff into the WKT header info?